.github/workflows/artifactLinks.yml

```diff
@@ -0,0 +1,20 @@
+name: Artifact links comments creator
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types: [completed]
+
+jobs:
+  artifacts-url-comments:
+    name: Add artifact links to PR and issues
+    runs-on: windows-2022
```

Comment:
Using Linux is usually faster/cheaper, or does this action require windows?

Reply:
I can try, but i think this script is using windows.

Reply:
It seems is simply a Node application, so is probably that it will work. I've found several repos that use it with Ubuntu, like this:
https://github.com/SyGuS-Org/docs/blob/master/.github/workflows/pr-comment.yaml

They seem to have worked, but the logs are missing (I suppose because they are old, more than 4 months)...